packages/bootstrap/scss/table/_layout.scss

```diff
@@ -0,0 +1,9 @@
+@import "~@progress/kendo-theme-default/scss/table/_layout.scss";
+
+@include exports("table/theme/bootstrap") {
+
+    .k-table-body .k-table-group-body,
```

Comment:
It seems that this code should be in the _theme.scss file, not in the _layout.scss.
Also, not sure where this color comes from but shouldn't it be in a variable instead of hardcoding it?